nibabel/loadsave.py

```diff
@@ -41,7 +41,11 @@ def load(filename, **kwargs):
     img : ``SpatialImage``
        Image of guessed type
     '''
-    return guessed_image_type(filename).from_filename(filename, **kwargs)
```

Comment:
For completeness, there are 11 standard extensions defined by CIFTI-2 that are in use, see Appendix A in the document:

http://www.nitrc.org/forum/attachment.php?attachid=342&group_id=454&forum_id=1955

There are also a few extensions (that are not in common use yet) that use the same mapping types, requiring a specific meaning on what is otherwise a "scalar" mapping, see the "Specializations of Scalar Maps" section near the end of the main document:

http://www.nitrc.org/forum/attachment.php?attachid=341&group_id=454&forum_id=1955